2021-12-03 09:36:31,438 [cuckoo.core.scheduler] INFO: Task #25: acquired machine cuckoo1 (label=cuckoo1)
2021-12-03 09:36:31,439 [cuckoo.core.resultserver] DEBUG: Now tracking machine 192.168.122.227 for task #25
2021-12-03 09:36:31,439 [cuckoo.core.plugins] DEBUG: Started auxiliary module: Replay
2021-12-03 09:36:31,445 [cuckoo.auxiliary.sniffer] INFO: Started sniffer with PID 37915 (interface=qemubr, host=192.168.122.227)
2021-12-03 09:36:31,446 [cuckoo.core.plugins] DEBUG: Started auxiliary module: Sniffer
2021-12-03 09:36:31,460 [cuckoo.machinery.qemu] DEBUG: Starting vm cuckoo1
2021-12-03 09:36:31,483 [cuckoo.machinery.qemu] DEBUG: Executing QEMU ['qemu-system-x86_64', '-display', 'none', '-m', '1024M', '-hda', '/home/crochetch/Documents/PhD/VMImages/images/snapshot_cuckoo1.qcow2', '-net', 'tap,ifname=tap_cuckoo1,script=no,downscript=no', '-net', 'nic,macaddr=52:54:00:12:34:56', '-enable-kvm']
2021-12-03 09:36:31,508 [cuckoo.core.guest] INFO: Starting analysis #25 on guest (id=cuckoo1, ip=192.168.122.227)
2021-12-03 09:36:31,514 [cuckoo.core.guest] INFO: Guest is running Cuckoo Agent 0.10 (id=cuckoo1, ip=192.168.122.227)
2021-12-03 09:36:31,517 [cuckoo.core.guest] ERROR: No valid analyzer found at path: /home/crochetch/Documents/toolchain_malware_analysis/src/sandboxes/.cuckoo/analyzer/:platform:
2021-12-03 09:36:31,518 [cuckoo.core.scheduler] ERROR: Error from the Cuckoo Guest: No valid analyzer found for :platform: platform!
2021-12-03 09:36:31,523 [cuckoo.core.plugins] DEBUG: Stopped auxiliary module: Replay
2021-12-03 09:36:31,523 [cuckoo.core.plugins] ERROR: Unable to stop auxiliary module: Sniffer
Traceback (most recent call last):
  File "/home/crochetch/Documents/toolchain_malware_analysis/penv-2.7/lib/python2.7/site-packages/Cuckoo-2.0.7-py2.7.egg/cuckoo/core/plugins.py", line 164, in stop
    module.stop()
  File "/home/crochetch/Documents/toolchain_malware_analysis/penv-2.7/lib/python2.7/site-packages/Cuckoo-2.0.7-py2.7.egg/cuckoo/auxiliary/sniffer.py", line 156, in stop
    (out, err, faq("permission-denied-for-tcpdump"))
CuckooOperationalError: Error running tcpdump to sniff the network traffic during the analysis; stdout = '' and stderr = 'tcpdump: qemubr: No such device exists\n(SIOCGIFHWADDR: No such device)\n'. Did you enable the extra capabilities to allow running tcpdump as non-root user and disable AppArmor properly (the latter only applies to Ubuntu-based distributions with AppArmor, see also https://cuckoo.sh/docs/faq/index.html#permission-denied-for-tcpdump)?
2021-12-03 09:36:31,528 [cuckoo.machinery.qemu] DEBUG: Stopping vm cuckoo1
2021-12-03 09:36:31,539 [cuckoo.core.resultserver] DEBUG: Stopped tracking machine 192.168.122.227 for task #25
2021-12-03 09:36:31,549 [cuckoo.core.scheduler] DEBUG: Released database task #25
2021-12-03 09:36:31,568 [cuckoo.core.plugins] DEBUG: Executed processing module "AnalysisInfo" for task #25
2021-12-03 09:36:31,569 [cuckoo.processing.behavior] WARNING: Analysis results folder does not contain any behavior log files.
2021-12-03 09:36:31,569 [cuckoo.core.plugins] DEBUG: Executed processing module "BehaviorAnalysis" for task #25
2021-12-03 09:36:31,570 [cuckoo.core.plugins] DEBUG: Executed processing module "Dropped" for task #25
2021-12-03 09:36:31,570 [cuckoo.core.plugins] DEBUG: Executed processing module "DroppedBuffer" for task #25
2021-12-03 09:36:31,570 [cuckoo.core.plugins] DEBUG: Executed processing module "MetaInfo" for task #25
2021-12-03 09:36:31,571 [cuckoo.core.plugins] DEBUG: Executed processing module "ProcessMemory" for task #25
2021-12-03 09:36:31,571 [cuckoo.core.plugins] DEBUG: Executed processing module "Procmon" for task #25
2021-12-03 09:36:31,572 [cuckoo.core.plugins] DEBUG: Executed processing module "Screenshots" for task #25
2021-12-03 09:36:31,572 [cuckoo.core.plugins] DEBUG: Executed processing module "Static" for task #25
2021-12-03 09:36:31,578 [cuckoo.core.plugins] DEBUG: Executed processing module "Strings" for task #25
2021-12-03 09:36:31,583 [cuckoo.core.plugins] DEBUG: Executed processing module "TargetInfo" for task #25
2021-12-03 09:36:31,583 [cuckoo.processing.network] WARNING: The PCAP file does not exist at path "/home/crochetch/Documents/toolchain_malware_analysis/src/sandboxes/.cuckoo/storage/analyses/25/dump.pcap".
2021-12-03 09:36:31,583 [cuckoo.core.plugins] DEBUG: Executed processing module "NetworkAnalysis" for task #25
2021-12-03 09:36:31,584 [cuckoo.core.plugins] DEBUG: Executed processing module "Extracted" for task #25
2021-12-03 09:36:31,584 [cuckoo.core.plugins] DEBUG: Executed processing module "TLSMasterSecrets" for task #25
2021-12-03 09:36:31,585 [cuckoo.processing.debug] ERROR: Error processing task #25: it appears that the Virtual Machine hasn't been able to contact back to the Cuckoo Host. There could be a few reasons for this, please refer to our documentation on the matter: https://cuckoo.sh/docs/faq/index.html#troubleshooting-vm-network-configuration
2021-12-03 09:36:31,591 [cuckoo.core.plugins] DEBUG: Executed processing module "Debug" for task #25
2021-12-03 09:36:31,592 [cuckoo.core.plugins] DEBUG: Running 0 signatures
2021-12-03 09:36:31,593 [cuckoo.core.plugins] DEBUG: Executed reporting module "JsonDump"
2021-12-03 09:36:31,620 [cuckoo.core.plugins] DEBUG: Executed reporting module "SingleFile"
2021-12-03 09:36:31,624 [cuckoo.core.plugins] DEBUG: Executed reporting module "MongoDB"
2021-12-03 09:36:31,624 [cuckoo.core.scheduler] INFO: Task #25: reports generation completed
2021-12-03 09:36:31,628 [cuckoo.core.scheduler] INFO: Task #25: analysis procedure completed
